test/e2e/crcsuite/crcsuite.go

```diff
@@ -112,11 +112,11 @@ func FeatureContext(s *godog.Suite) {
 		}
 
 		// remove $HOME/.crc
-		err = util.RemoveCRCHome(CRCHome)
-		if err != nil {
-			fmt.Println(err)
-			os.Exit(1)
-		}
+		// err = util.RemoveCRCHome(CRCHome)
```

Comment:
Is it wanted ?

Reply:
no no I just realize, I was undoing :), but this need to be reviewed as now is failing on windows, because if crc-admin-helper process is running this remove fails (just commented to test on windows) but forget undo 

Thx though!